File: testprojects/java/benchmark/accuracy-test-2013-04-17/technology/direct/dao/CallInstanceOuterClassDAO.java
<<<<<<< HEAD
package technology.direct.dao;

public class CallInstanceOuterClassDAO {
    // create an array
    private final static int SIZE = 15;
    private int[] arrayOfInts = new int[SIZE];
    
    public CallInstanceOuterClassDAO() {
        // fill the array with ascending integer values
        for (int i = 0; i < SIZE; i++) {
            arrayOfInts[i] = i;
        }
    }
    
    public void printEven() {
        // print out values of even indices of the array
        CallInstanceInnerClassDAO iterator = this.new CallInstanceInnerClassDAO();
        while (iterator.hasNext()) {
            System.out.println(iterator.getNext() + " ");
        }
    }
    
    // inner class implements the Iterator pattern
    public class CallInstanceInnerClassDAO {
        // start stepping through the array from the beginning
        private int next = 0;
        
        public boolean hasNext() {
            return true ;
        }
        
        public int getNext() {
            // record a value of an even index of the array
            int retValue = arrayOfInts[next];
            //get the next even element
            next += 2;
            return retValue;
        }
    }
=======
package technology.direct.dao;

public class CallInstanceOuterClassDAO {
    // create an array
    private final static int SIZE = 15;
    private int[] arrayOfInts = new int[SIZE];
    
    public CallInstanceOuterClassDAO() {
        // fill the array with ascending integer values
        for (int i = 0; i < SIZE; i++) {
            arrayOfInts[i] = i;
        }
    }
    
    public void printEven() {
        // print out values of even indices of the array
        CallInstanceInnerClassDAO iterator = this.new CallInstanceInnerClassDAO();
        while (iterator.hasNext()) {
            System.out.println(iterator.getNext() + " ");
        }
    }
    
    // inner class implements the Iterator pattern
    public class CallInstanceInnerClassDAO {
        // start stepping through the array from the beginning
        private int next = 0;
        
        public boolean hasNext() {
            return true ;
        }
        
        public int getNext() {
            // record a value of an even index of the array
            int retValue = arrayOfInts[next];
            //get the next even element
            next += 2;
            return retValue;
        }
    }
>>>>>>> 7b87626b99107db400440335a38d325b9ac19cd3
}

File: testresources/csharp/benchmark/Technology/Direct/Dao/CallInstanceOuterClassDAO.cs
namespace Technology.Direct.Dao
{
	public class CallInstanceOuterClassDAO
	{
		// create an array
        private const int SIZE = 7;
        private int[] arrayOfInts = new int[SIZE];

		public CallInstanceOuterClassDAO()
		{
			// fill the array with ascending integer values
			for (int i = 0; i < SIZE; i++)
			{
				arrayOfInts[i] = i;
			}
		}

		public virtual void PrintEven()
		{
			// print out values of even indices of the array
			CallInstanceOuterClassDAO.CallInstanceInnerClassDAO iterator = new CallInstanceOuterClassDAO.CallInstanceInnerClassDAO
				(this);
			while (iterator.HasNext())
			{
				System.Console.Out.WriteLine(iterator.GetNext() + " ");
			}
		}

        // inner class implements the Iterator pattern
		public class CallInstanceInnerClassDAO
		{
			private readonly CallInstanceOuterClassDAO _enclosing;
			private int next = 0;

			// start stepping through the array from the beginning
			public virtual bool HasNext()
			{
				return true;
			}

			public virtual int GetNext()
			{
				// record a value of an even index of the array
				int retValue = this._enclosing.arrayOfInts[this.next];
				//get the next even element
				this.next += 2;
				return retValue;
			}

			internal CallInstanceInnerClassDAO(CallInstanceOuterClassDAO _enclosing)
			{
				this._enclosing = _enclosing;
                
			}
		}
	}
}
